tools/c7n_tencentcloud/c7n_tencentcloud/resources/mysql_backup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n_tencentcloud.provider import resources
+from c7n_tencentcloud.query import ResourceTypeInfo, QueryResourceManager
+from c7n_tencentcloud.utils import PageMethod, isoformat_datetime_str
+import pytz
+
+
+@resources.register("mysql-backup")
+class MySQLBackUp(QueryResourceManager):
+    """mysql-backup"""
+
+    class resource_type(ResourceTypeInfo):
+        """resource_type"""
+        id = "InstanceId"
+        endpoint = "cdb.tencentcloudapi.com"
+        service = "cdb"
+        version = "2017-03-20"
+        enum_spec = ("DescribeDBInstances", "Response.Items[]", {})
```

Comment:
this seems problematic but given that there doesnt seem to be any other way to get backups not associated with an instance (e.g. instance was deleted). I see that the console itself is querying for backups this way:

```
curl 'https://intl-db-api.tencentcloud.com/api/server-cloud3/supeross/service.backup.DescribeBackups' \
-X 'POST' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-H 'Accept-Language: en-US,en;q=0.9' \
-H 'Accept-Encoding: gzip, deflate, br' \
-H 'Origin: https://console.tencentcloud.com' \
-H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.3 Safari/605.1.15' \
-H 'Referer: https://console.tencentcloud.com/mysql/backup/list/data' \
-H 'Content-Length: 168' \
-H 'Connection: keep-alive' \
-H 'Host: intl-db-api.tencentcloud.com' \
-H 'X-SeqId: 093e6a80-594e-11ed-92d8-ef39972cfd7e' \
-H 'X-CsrfCode: 1366196663' \
-H 'X-Requested-With: XMLHttpRequest' \
-H 'X-QCloud-User-ID: 1307793390' \
-H 'X-QCloud-Transaction-ID: 093e6a80-594e-11ed-92d8-ef39972cfd7e' \
--data-binary '{"Region":"na-ashburn","Limit":20,"Offset":0,"ApiModule":"backupService","Product":"mysql","OrderBy":"Date","OrderDirection":"DESC","MaxStartTime":"","MinStartTime":""}'
```

Reply:
After confirmation, the api is like this, if the instance has been deleted, the backup cannot be queried

Reply:
Are you able to open an upstream ticket to get that type of functionality? For now it's ok, given the constraints of the API but this doesn't really behave like any of the other cloud providers where backups can be queried without a parent resource

Reply:
A requirement has been submitted. The product team follow it. 
I think we can merge this version and optimize it later when the new API is ready.